server/datastore/mysql/migrations/tables/20230807100822_AddIndexNanoEnrollment.go

```diff
@@ -0,0 +1,19 @@
+package tables
+
+import (
+	"database/sql"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230807100822, Down_20230807100822)
+}
+
+func Up_20230807100822(tx *sql.Tx) error {
```

Comment:
can we make a note as we did with the other migrations stating that this is coming from the nano repo?